src/figuresClasses.ts

```diff
@@ -1,19 +1,82 @@
+type Color = 'red' | 'blue' | 'green';
+type Shape = 'circle' | 'rectangle' | 'triangle';
+
 export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
 
+export function roundNum(num : number) :number {
+  return Math.floor(num * 100) / 100;
 }
 
-export class Triangle {
+export class Circle implements Figure {
+  shape: Shape = 'circle';
 
-}
+  constructor(
+    public color: Color,
+    public radius: number,
+  ) {
+    if (radius <= 0) {
+      throw new Error('Radius cant be 0 or less');
```

Comment:
You can add one single quote like this `\'`

Reply:
yeah, thanks - just missed, that i need an apostrophe)